src/networking.c

```diff
@@ -890,6 +896,13 @@ void addReplyBulkSds(client *c, sds s)  {
     addReply(c,shared.crlf);
 }
 
+/* Similar with addReplyBulkSds but doesn't free the SDS string. */
+void addReplyBulkSdsNoFree(client *c, sds s)  {
```

Comment:
Why not use `addReplyBulkCBuffer`?

Reply:
ohh.. right i forget this one, yes i can use it